docs/faq/loading/Broker_load_faq.md

```diff
@@ -1,27 +1,96 @@
 
-Column names on both sides are not consistent in the source file and the StarRocks. Type inference within the system was performed on Set and failed on Cast. To resolve this, you can change the field names on both sides to be identical. After that, Set or Cast is not needed, thus making import successful.
+## 5. What do I do if the "failed to send batch" or "TabletWriter add batch with unknown id" error occurs?
 
-A successfully executed cannot be executed again. To execute the task again, you need to reinitiate it. This is because the imported labels are not reusable, avoiding loss or duplication of imported tasks. You can use show load to see the import history, then copy it, modify the label and re-import it.
+The amount of time taken to write the data exceeds the upper limit, causing a timeout error. To resolve this issue, modify the settings of the [session variable](../../reference/System_variable.md) `query_timeout` and the [BE configuration item](../../administration/Configuration.md#configure-be-static-parameters) `streaming_load_rpc_max_alive_time_sec` based on your business requirements.
 
-## No error is reported in Broker Load, but the data still cannot be queried
+## 6. What do I do if the "LOAD-RUN-FAIL; msg:OrcScannerAdapter::init_include_columns. col name = xxx not found" error occurs?
 
-Broker Load is asynchronous. Successfully creating load statement does not guarantee successful import. You can use show load to view the current status and errmsg. Modify related contents and retry the task.
+If you are loading Parquet- or ORC-formatted data, check whether the column names held in the first row of the source data file are the same as the column names of the destination StarRocks table.
 
-Column names on both sides are not consistent in the source file and the StarRocks. Type inference within the system was performed on Set and failed on Cast. To resolve this, you can change the field names on both sides to be identical. After that, Set or Cast is not needed, thus making import successful.
+```SQL
+(tmp_c1,tmp_c2)
+SET
+(
+   id=tmp_c2,
+   name=tmp_c1
+)
+```
+
+The preceding example maps the `tmp_c1` and `tmp_c2` columns of the source data file onto the `name` and `id`  columns of the destination StarRocks table, respectively. If you do not specify the `SET` clause, the column names specified in the `column_list` parameter are used to declare the column mapping. For more information, see [BROKER LOAD](../../sql-reference/sql-statements/data-manipulation/BROKER%20LOAD.md).
+
+> **NOTICE**
+>
+> If the source data file is an ORC-formatted file generated by Apache Hive™ and the first row of the file holds `(_col0, _col1, _col2, ...)`, the "Invalid Column Name" error may occur. If this error occurs, you need to use the `SET` clause to specify the column mapping.
+
+## 7. How do I handle errors such as the error due to which the Broker Load job remains running for a long period of time and cannot finish?
```

Comment:
such as the error that causes the Broker Load job to run for an excessively long period of time and cannot finish?